packages/rocketchat-lib/server/methods/unblockUser.js

```diff
@@ -0,0 +1,22 @@
+Meteor.methods({
+	unblockUser({rid, blocked}) {
+
+		check(rid, String);
+		check(blocked, String);
+
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'blockUser' });
+		}
+
+		const subscription = RocketChat.models.Subscriptions.findOneByRoomIdAndUserId(rid, Meteor.userId());
+		const subscription2 = RocketChat.models.Subscriptions.findOneByRoomIdAndUserId(rid, blocked);
+
+		if (!subscription || !subscription2) {
+			throw new Meteor.Error('error-invalid-room', 'Invalid room', { method: 'blockUser' });
+		}
+
+		RocketChat.models.Subscriptions.unsetBlockedByRoomId(rid, blocked, Meteor.userId());
```

Comment:
Because I need to determine who blocked and who got blocked. I could use the room object for that, but then I'd be storing the blocker user _id and the blocked user _id. Any reason why setting room would be better than subscription?